src/module.d.ts

```diff
@@ -510,6 +510,35 @@ declare class posthog {
 
     /* Will log all capture requests to the Javascript console, including event properties for easy debugging */
     static debug(): void
+
+    /*
+     * Starts session recording and updates disable_session_recording to false.
+     * Used for manual session recording management. By default, session recording is enabled and
+     * starts automatically.
+     *
+     * ### Usage:
+     *
+     *     posthog.startSessionRecording()
+     */
+    static startSessionRecording(): void
```

Comment:
what other type changes do you mean? @macobo

Reply:
Looks good now. I might have been looking at a cached version or smth, last time it wasn't here.